### PR TITLE
Initial scaffolding 

### PR DESCRIPTION
**Pull request by rnugraha**
Scaffold the project with main tools and frameworks. There are mainly two main parts of frameworks and tools installed:

##### Tools 
List of packages to support the project, listed in package.json
- Gulp 
- Bower 
- Jasmin
- Karma
- BrowserSync
- Protractor

##### UI Frameworks 
- AngularJS
- Restangular
- Bootstrap
- Angular UI Bootstrap
- Less

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <body>
+    <!--[if lt IE 10]>
+      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+
+    <div ui-view></div>
+
+    <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
+    <script>
+      (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
```
Comment (kretep) [inline src/index.html]: We're going to track users? ;)

**Comment by kretep** [comment]
Looks good! (The favicon could use some work though :P)

File: src/index.html (hunk 1 above)
Comment (rnugraha) [inline src/index.html]: It was provided by the generator and I would say it customizable. Can be removed whenever it's not needed, but also giving the possibility for those who want to have it.

Comment (Sjoerd-van-Hagen) [inline src/index.html]: Better remove it right away.

**Comment by Sjoerd-van-Hagen** [comment]
Holy crap! What happened to just writing some html, js and css? This seems rather convoluted.

**Comment by rnugraha** [comment]
@Sjoerd-van-Hagen This might a good start for you http://blog.chartbeat.com/2014/01/30/modern-front-end-workflow-start-finish/

**Comment by Sjoerd-van-Hagen** [comment]
Thanks. Could you put a line in the readme to point aspiring developers to that page? Do you use all the practices in that page? If not could you also point out where you diverge from it and why?

**Comment by rnugraha** [comment]
@Sjoerd-van-Hagen : should be provided on the second pull request. Will merge this there's no further critical issues.

**Pull request merged**